src/cbapi/response/models.py

```diff
@@ -70,6 +70,43 @@
 #     urlobject = '/api/v1/binary'
 
 
+class StoragePartitionQuery(SimpleQuery):
+    @property
+    def results(self):
+        if not self._full_init:
+            self._results = []
+            for k,v in iteritems(self._cb.get_object(self._urlobject, default={})):
+                t = self._doc_class.new_object(self._cb, v, full_doc=True)
+                if self._match_query(t):
+                    self._results.append(t)
+            self._results = self._sort(self._results)
+            self._full_init = True
+
+        return self._results
+
+
+class StoragePartition(NewBaseModel):
+    urlobject = "/api/v1/storage/events/partition"
+    primary_key = "name"
+
+    @classmethod
+    def _query_implementation(cls, cb):
+        return StoragePartitionQuery(cls, cb)
+
+    def _refresh(self):
+        # there is no GET method for a StoragePartition.
+        return True
+
+    def delete(self):
```

Comment:
Would it make sense to describe these APIs here?
I would use descriptions from the confluence page https://confluence.carbonblack.local/display/CBENT/API+Changes+for+Time+Partitioning